### ISSUE TITLE
Gene symbols should be in italic

### ISSUE THREAD

**Issue by sandrine-m** [NCATSTranslator/Feedback#300]
According to rules on Gene names, it would be good to show them in italic.

**Comment by LEHunter** [comment]
While it is true that italics have often used to distinguish between genes (italicized) and proteins, this is not a universal practice. Confusingly, genes from different organisms are also sometimes distinguished by italics  There are guidelines that conflict, e.g.,

<https://www.jci.org/kiosk/publish/genestyle>
Gene Nomenclature and Style<https://www.jci.org/kiosk/publish/genestyle>
jci.org<https://www.jci.org/kiosk/publish/genestyle>
[jci-favicon-378ea4dd43f03bc78136c6a261d9b28f6811fe59d12db426eae78cf0691b0008.ico]<https://www.jci.org/kiosk/publish/genestyle>

https://www.biosciencewriters.com/Guidelines-for-Formatting-Gene-and-Protein-Names.aspx

<https://en.wikipedia.org/wiki/Gene_nomenclature>
Gene nomenclature<https://en.wikipedia.org/wiki/Gene_nomenclature>
en.wikipedia.org<https://en.wikipedia.org/wiki/Gene_nomenclature>
[wikipedia.png]<https://en.wikipedia.org/wiki/Gene_nomenclature>

There are also standards for capitalization, etc.


On Jun 6, 2023, at 11:46 AM, sandrine-m ***@***.***> wrote:

[External Email - Use Caution]


According to rules on Gene names, it would be good to show them in italic.

**Comment by sandrine-m** [comment]
thank you @LEHunter for the material! Do you mean common gene names or gene symbols? I just realized I made the "conflation" while typing the issue in and corrected into Gene symbols instead of common gene names...

**Comment by LEHunter** [comment]
Yes, it is widely agreed that only gene symbols are italicized, never full gene names.

On Jun 6, 2023, at 1:00 PM, sandrine-m ***@***.***> wrote:

[External Email - Use Caution]


thank you @LEHunter<https://github.com/LEHunter> for the material! Do you mean common gene names or gene symbols? I just realized I made the "conflation" while typing the issue in and corrected into Gene symbols instead of common gene names...

**Comment by sstemann** [comment]
@dnsmith124 @desiyopp is this something we can do now?

**Comment by Genomewide** [comment]
@dnsmith124 I think it would mean on the autocomplete, and any node with category of gene. We would need to check that they gene node names are always symbols and not gene names.  That would make it the most simple.

**Comment by dnsmith124** [comment]
@Genomewide I feel like I need some more information before I can implement this effectively. I'm not certain how I can programmatically confirm that the label for a node of type `biolink:Gene` is a symbol and not a name, can you shed some light on the conventions used here?

**Comment by Genomewide** [comment]
Sorry, that is actually the issue.  I think that we need to ask the data modeling people if that is a standard. I think it is, but am not sure it is always that way.  If it is not, then I don't think we can do it.

**Comment by sierra-moxon** [comment]
`biolink:Gene` has both attributes, `biolink:name` and a `biolink:symbol` 

@LEHunter is right, nomenclature conventions for the gene symbol (`biolink:symbol`) are different and often conflict between organisms.  

But we can follow the nomenclature guidelines for italics and capitalization for `biolink:symbol` based on the following official guidelines from established model organism databases and HGNC.  

Human: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7494048/ 
Saccharomyces: https://sites.google.com/view/yeastgenome-help/community-help/nomenclature-conventions
Fission yeast: https://www.pombase.org/submit-data/gene-naming-guidelines
Worm: https://wormbase.org//about/userguide/nomenclature#6h9a7ic1jm02g58ebkd34fl--10
Fly: https://wiki.flybase.org/wiki/FlyBase:Nomenclature#Gene_symbols_and_names
Zebrafish: https://zfin.atlassian.net/wiki/spaces/general/pages/1818394635/ZFIN+Zebrafish+Nomenclature+Conventions#ZFINZebrafishNomenclatureConventions-1
Frog: https://www.xenbase.org/gene/static/geneNomenclature.jsp
Mouse: https://www.informatics.jax.org/mgihome/nomen/gene.shtml#gene_sym
Rat: https://rgd.mcw.edu/nomen/nomen.shtml

Granted, this is a LOT of info to digest, and none of it is formalized or even explained in Biolink.  I can take it as an action item to try and provide some of it in Biolink.  I can say that folks tend to notice right away when they see gene **_symbol_** nomenclature not matching the reported taxon-specific nomenclature requirements of the gene.

**Comment by sierra-moxon** [comment]
@dnsmith124 - should we close this as "won't do"? or is it tractable as a priority?

**Comment by dnsmith124** [comment]
@sierra-moxon personally I think we should close this as "won't do", given the massive amount of work required to implement this and the relatively small benefit.
